string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,16 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var lowerCase = word.toLowerCase();
+  var seperateWord = lowerCase.slice(1, 4);
+  var firstLetter = word[0];
+  var upperCase = firstLetter.toUpperCase();
+  if (lowerCase === 'javascript') {
+    var remainderOfWord = lowerCase.slice(5, 10);
+    var specialCase = word[4];
+    var specialUpperCase = specialCase.toUpperCase();
+    var newWord = upperCase + seperateWord + specialUpperCase + remainderOfWord;
```

Comment:
Sorry, I was not very clear in my last comment. I meant if the lowercased word is `'javascript'`, then just `return 'JavaScript'`. You don't have to slice it up if your code "knows" what word it is.